open5gs/changelog.md

```diff
@@ -4,6 +4,7 @@
 ### Changed
```

Comment:
Please @bjoern-r increase the version number of the component into the changelog